src/components/elements/link.tsx

```diff
@@ -44,7 +44,7 @@ const DrupalLink = ({href, children, ...props}: Props) => {
   }
 
   return (
-    <Link href={href} {...props} className={twMerge("text-stone-dark hocus:text-archway-dark", props.className)}>
+    <Link href={href} className={clsx("text-digital-red *:text-digital-red hocus:text-archway-dark *:hocus:text-archway-dark", props.className)} {...props}>
```

Comment:
what does `*:` do?

Reply:
Direct children styling! [TW Docs for ref](https://tailwindcss.com/docs/hover-focus-and-other-states#styling-direct-children) 😄